cmd_migrate_db.go

```diff
@@ -0,0 +1,650 @@
+					"bitcoin", network,
+				),
+			)
+		}
+
+		// Path to the db file.
+		var path string
+		switch prefix {
+		case lncfg.NSChannelDB:
+			path = filepath.Join(graphDir, lncfg.ChannelDBName)
+
+		case lncfg.NSMacaroonDB:
+			path = filepath.Join(walletDir, lncfg.MacaroonDBName)
+
+		case lncfg.NSDecayedLogDB:
+			path = filepath.Join(graphDir, lncfg.DecayedLogDbName)
+
+		case lncfg.NSTowerClientDB:
+			path = filepath.Join(graphDir, lncfg.TowerClientDBName)
+
+		case lncfg.NSTowerServerDB:
+			path = filepath.Join(
+				towerServerDir, lncfg.TowerServerDBName,
+			)
+
+		case lncfg.NSWalletDB:
+			path = filepath.Join(walletDir, lncfg.WalletDBName)
+		}
+
+		const (
+			noFreelistSync = true
+			timeout        = time.Minute
+		)
+
+		args = []interface{}{
+			path, noFreelistSync, timeout,
+		}
+		backend = kvdb.BoltBackendName
+		logger.Infof("Opening bolt backend at %s for prefix '%s'",
+			path, prefix)
+
+	case kvdb.PostgresBackendName:
+		args = []interface{}{
+			context.Background(),
+			&postgres.Config{
+				Dsn:            cfg.Postgres.Dsn,
+				Timeout:        time.Minute,
+				MaxConnections: 10,
+			},
+			prefix,
+		}
+
+		logger.Infof("Opening postgres backend at %s with prefix '%s'",
+			cfg.Postgres.Dsn, prefix)
+
+	case kvdb.SqliteBackendName:
+		// Directories where the db files are located.
+		graphDir := lncfg.CleanAndExpandPath(
+			filepath.Join(cfg.Sqlite.DataDir, "graph", network),
+		)
+		walletDir := lncfg.CleanAndExpandPath(
+			filepath.Join(
+				cfg.Sqlite.DataDir, "chain", "bitcoin", network,
+			),
+		)
+
+		// In case the data directory was set but the watchtower is
+		// still the default one, we use the data directory for the
+		// watchtower as well.
+		towerServerDir := lncfg.CleanAndExpandPath(
+			filepath.Join(
+				cfg.Sqlite.TowerDir, "watchtower", "bitcoin",
+				network,
+			),
+		)
+		if cfg.Sqlite.DataDir != defaultDataDir &&
+			cfg.Sqlite.TowerDir == defaultDataDir {
+
+			towerServerDir = lncfg.CleanAndExpandPath(
+				filepath.Join(
+					cfg.Sqlite.DataDir, "watchtower",
+					"bitcoin", network,
+				),
+			)
+		}
+
+		var dbName string
+		var path string
+		switch prefix {
+		case lncfg.NSChannelDB:
+			path = graphDir
+			dbName = lncfg.SqliteChannelDBName
+		case lncfg.NSMacaroonDB:
+			path = walletDir
+			dbName = lncfg.SqliteChainDBName
+
+		case lncfg.NSDecayedLogDB:
+			path = graphDir
+			dbName = lncfg.SqliteChannelDBName
+
+		case lncfg.NSTowerClientDB:
+			path = graphDir
+			dbName = lncfg.SqliteChannelDBName
+
+		case lncfg.NSTowerServerDB:
+			path = towerServerDir
+			dbName = lncfg.SqliteChannelDBName
+
+		case lncfg.NSWalletDB:
+			path = walletDir
+			dbName = lncfg.SqliteChainDBName
+
+		case lncfg.NSNeutrinoDB:
+			dbName = lncfg.SqliteNeutrinoDBName
+		}
+
+		args = []interface{}{
+			context.Background(),
+			&sqlite.Config{
+				Timeout: time.Minute,
+			},
+			path,
+			dbName,
+			prefix,
+		}
+
+		logger.Infof("Opening sqlite backend with prefix '%s'", prefix)
```

Comment:
Can we also log the `path` + `dbName` here? So it's more easy to see in the log what bolt DBs are merged into which SQLite DBs?